src/main/java/org/broadinstitute/consent/http/db/InstitutionDAO.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.models.Institution;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.sqlobject.transaction.Transactional;
+
+import java.util.Date;
+import java.util.List;
+
+public interface InstitutionDAO extends Transactional<InstitutionDAO> {
+
+  @SqlUpdate("insert into institution " +
+    "(institution_name, it_director_name, it_director_email, create_user, create_date) " +
+    "values (:institutionName, :itDirectorName, :itDirectorEmail, :createUser, :createDate)")
+  @GetGeneratedKeys
+  Integer insertInstitution(@Bind("institutionName") String institutionName,
+                            @Bind("itDirectorName") String itDirectorName,
+                            @Bind("itDirectorEmail") String itDirectorEmail,
+                            @Bind("createUser") Integer createUser,
+                            @Bind("createDate") Date createDate);
+
+  @SqlUpdate("update institution set" +
+    " institution_id = :institutionId, " +
+    " institution_name = :institutionName," +
+    " it_director_name = :itDirectorName," +
+    " it_director_email = :itDirectorEmail," +
+    " update_user = :updateUser," +
+    " update_date = :updateDate" +
+    " where institution_id = :institutionId")
```

Comment:
same feedback as above applies to all of the sql, but one more helpful hint. I like to have a space after the starting quote and before the ending quote for safety. It's very easy to mistakenly write:
```
    "update_date = :updateDate" +
    "where institution_id = :institutionId")
```
which ends up becoming: `update_date = :updateDatewhere institution_id = :institutionId`
Adding spaces liberally in the string construction is helpful.